components/service/profile/ProfileModal/ProfileModal.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import { useState } from 'react';
+import { Checkbox } from '@/components/ui/checkbox';
+import JobSelect from '@/components/service/profile/ProfileModal/JobSelectbar';
```

Comment:
```suggestion
import JobSelectbar from '@/components/service/profile/ProfileModal/JobSelectbar';
```

@Kristyn-Yoon 
실제 컴포넌트 이름과 동일하게 맞춰주세요!